pkg/skaffold/kubectl/cli.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
```

Comment:
I found myself often doing named imports like
```go
import kubectlcli "(..)/pkg/skaffold/kubectl"
```
To circumvent name clashes with other `kubectl` packages. Should I rename this package to avoid these problems?

Reply:
I'd rather keep it `kubectl` as this _is_ the main kubectl package, and specify the clashing ones instead as `deploy` or similar.